recognition/include/pcl/recognition/dense_quantized_multi_mod_template.h

```diff
@@ -66,6 +67,12 @@ namespace pcl
 
       std::size_t num_of_features;
       read (stream, num_of_features);
+      PCL_CHECK_IO_STREAM(stream, "num_of_features");
+      if (num_of_features <= 0)
+      {
+        PCL_THROW_EXCEPTION(pcl::IOException, "Error! Number of features read from file is not positive!");
```

Comment:
Some of the messages can be reworded to be less ambiguous. This message here can mean that the total number of features read from file are not positive.

Perhaps 'Number of features specified in the file is not positive' and similar are a good alternative.